kiwi/models/predictor_estimator.py

```diff
@@ -85,7 +85,7 @@ def __init__(
 
 @Model.register_subclass
 class Estimator(Model):
-    title = 'PredEst (Predictor-Estimator)'
+    title = "PredEst (Predictor-Estimator)"
```

Comment:
Is there a reason for preferring double quotes in these cases? Overall the codebase seems to be inconsistent with regard to this, with single quotes being used more frequently.

My personal preference is to use single quotes except for multiline strings ( """ """ ) and strings that contain single quotes ( " ' " )

Reply:
Yes, this was an issue with the black formatter. I've pushed a new version with his corrected! 👍